extensions/amp-iframe/amp-iframe.md

```diff
@@ -84,3 +84,27 @@ Here are some factors that affect how fast the resize will be executed:
 - Whether the resize is triggered by the user action;
 - Whether the resize is requested for a currently active IFrame;
 - Whether the resize is requested for an IFrame below the viewport or above the viewport.
+
+#### Iframe Click-To-Play
+It is possible to have an `amp-iframe` appear on the top of a document when the `amp-ifame` has a `placeholder` element as shown in the example below.
+
+```html
+<amp-iframe width=300 height=300
+   layout="responsive"
+   sandbox="allow-scripts"
+   resizable
+   src="https://foo.com/iframe">
+ <amp-img layout="fill" src="https://foo.com/foo.png" placeholder></amp-img>
+</amp-iframe>
+```
+- The `amp-iframe` must contain an element with the `placeholder` attribute, preferably an `amp-img` which would be rendered as a placeholder till the iframe is ready to be displayed.
```

Comment:
I don't think we should be advising directly on `amp-img`. Perhaps, phrase it "for instance an amp-img element".